scripts/test.sh

```diff
@@ -25,51 +25,19 @@ function logmsg {
     echo -e "\n\n*** $1 ***\n"
 }
 
-function _gofmt {
-    logmsg "Running 'gofmt' ..."
-    test -z "$(gofmt -l -d ${SUBDIRS} | tee /dev/stderr)"
+function _gometalinter {
+    logmsg "Running 'gometaliner' ..."
+    gometalinter  --config=.gometalinter.json
```

Comment:
It looks like I missed couple dots here. It's fixed here https://github.com/dcos/dcos-diagnostics/pull/45